cms_backend/settings.py

```diff
@@ -32,13 +32,12 @@
 
 INSTALLED_APPS = [
     'django.contrib.admin',
-    'users.apps.UsersConfig'
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'django_graphene'
```

Comment:
Why did you delete graphene?

Reply:
@partha2000  the command to run django runsrever failed

see the attached SS  below:
![image](https://user-images.githubusercontent.com/60546840/104282258-a0ba6e00-5463-11eb-8b42-88bd115ad8cb.png)

Reply:
The `requirements.txt` was most probably missing graphene. It has been updated. Just do this to solve this error: `pip install -r requirements.txt`.

Reply:
ok understood. i will pull n update my pr